tests/Integration/ParserTest.php

```diff
@@ -86,12 +86,10 @@ public function testParseFile()
      *      Missing catalog.
      *
      * @see https://github.com/smalot/pdfparser/issues/267
+     * @doesNotPerformAssertions
```

Comment:
The purpose of this test (`testIssue267`) was to keep track of a faulty functionality. After your fix this test should be altered and actually do something with the extracted data. I am open for suggestions.

Reply:
Ok i will add some test cases.

Reply:
Looking forward to it.